extensions/airship/airship-event-extension.schema.json

```diff
@@ -0,0 +1,271 @@
+{
+  "$id": "https://ns.airship.com/airship-event-extension",
+  "title": "Airship Event Extension",
+  "description": "This mixin adds Airship specific event fields.",
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/experienceevent"
+  ],
+  "properties": {
+    "https://ns.airship.com/offset-id": {
```

Comment:
See my general comment. You probably want this to be named https://ns.adobe.com/airship/offset-id

Reply:
also i suggest to use offsetID instead of offset-id to be in consistent with other schema property names . Same applies for other property names